---
title: About
---

<style>
/**/
.introduction{display:inline-block; /*padding-right:10px;*/ padding-bottom:50px;}
.profile_photo_text{float:left; width: 35%;}
.introduction_text{padding-left:25px;float:right;width:65%}
.photo_text{float:left; width:90%; font-size: 12px; }
.link_text{float:left; width:90%; font-size: 14px; font-weight:bold;}
.introduction_text p {font-size: 14px;width:100%;}
/**/
/**/
.box{display: flex;vertical-align:top;}
.title_year{width:10%;font-weight: bold; /*padding-left:30px;*/ color: black;}
.Publications_text{font-size: 14px; padding-left:30px; width:90%;float:right;}
/**/
.footer{text-align: center;
		margin: 60px;
		}
</style>

<div class="introduction">
	<div class="profile_photo_text">
		<div class="images">
		<img src="./assets/images/lai_headshot.jpeg" width="314" height="392" />
		</div>
		<div class="photo_text">
			<p>Assistant Research Professor in Molecular Biology and Genetics.
				</br>
				Assistant Research Professor in Computational Biology.
				</br>
				458 Biotechnology Building, Cornell, Ithaca, NY, USA 14850</p>
		</div>
	</div>
	<div class="introduction_text">
			<h1>William Lai</h1>
			<P>The primary focus of my research program is to understand how protein-complexes target the genome, how protein binding changes across cell states, and how mis-regulation of these interactions can result in disease-phenotypes. I performed my graduate work at SUNY Buffalo, where I developed novel bioinformatic algorithms to analyze some of the first genomic datasets generated from high-throughput sequencing. I became fascinated with the shapes and patterns of the data and how they could be directly interpreted into biological function. During my postdoc at Penn State, I focused on developing novel high-resolution epigenomic technologies. I combined the algorithmic approaches I developed in graduate school with the epigenomic data generated during my postdoc to map the eukaryotic epigenome at ultra-high resolution. Developing and deploying these assays allowed me to generate thousands of new epigenomic datasets across human and yeast model systems.
			</br>			</br>
      I have recently come to Cornell University as an Assistant Research Professor as part of Cornell’s Genome Biology initiative. My group is now working closely with Weill Cornell Medical College to apply our epigenomic technology and algorithmic approaches to biomedical specimens to investigate the fundamental nature of human diseases. My research vision is to combine wet-bench biochemical genomic technology with analytical bioinformatic algorithmic approaches to understand how the DNA-binding patterns of protein-complexes reflect features of genome regulation and how this regulation differs in diseased patient samples.</P>
			<p>I believe that the greatest biological interpretations can be achieved using a combined biochemical and bioinformatic approach to integrating multiple types of high-resolution genomic data.</p>
	</div>
</div>



<div class="Publications">
	<h2>Publications</h2>
	<a href="/publications">View More</a>

	<div class="box">
		<div class="title_year">
			<p>2022</p>
		</div>
		<div class="Publications_text">
			<ul>
				<li><p><b><span style="color: black">ScriptManager: an interactive platform for reducing barriers to genomics analysis.</span></b><br />
				<span style="color:"><u>Lang O,</u> <u>Pugh BF,</u> <strong><u>Lai WKM.</u></strong> </span><br />
				<span style="color:black"><b>Practice and Experience in Advanced Research Computing</b></span> 2022, <a href="https://dl.acm.org/doi/fullHtml/10.1145/3491418.3535161">https://doi.org/10.1145/3491418.3535161</a> </p>

				<li><p><b><span style="color: black">An integrated SAGA and TFIID PIC assembly pathway selective for poised and induced promoters.</span></b><br />
				<span style="color:"><u>Mittal C,</u> <u>Lang O,</u> <strong><u>Lai WKM,</u></strong> <u>Pugh BF.</u> </span><br />
				<span style="color:black"><b>Genome Research</b></span> 2022, 36(17-18):985-1001. PMID: 36302553 PMCID: PMC9732905</p>

				<li><p><b><span style="color: black">Genome-wide promoter assembly in E. coli measured at single-base resolution.</span></b><br />
				<span style="color:"><u>John J,</u> <u>Jabbar J,</u> <u>Badjatia N,</u> <u>Rossi MJ,</u> <strong><u>Lai WKM,</u></strong> <u>Pugh BF.</u></span><br />
				<span style="color:black"><b>Genome Research</b></span> 2022, 32(5):878-892. PMID: 35483960 PMCID: PMC9104697</p>

				<li><p><b><span style="color: black">PEGR: a flexible management platform for reproducible epigenomic and genomic research.</span></b><br />
				<span style="color:"><u>Shao D,</u> <u>Kellogg G,</u> <u>Nematbakhsh A,</u> <u>Kuntala PK,</u> <u>Mahony S,</u> <u>Pugh BF,</u> <strong><u>Lai WKM.</u></strong></span><br />
				<span style="color:black"><b>Genome Biology</b></span> 2022, 19;23(1):99. PMID: 35440038 PMCID: PMC9016988</p>

				<li><p><b><span style="color: black">STENCIL: A web templating engine for visualizing and sharing life science datasets.</span></b><br />
				<span style="color:"><u>Sun Q,</u> <u>Nematbakhsh A,</u> <u>Kuntala PK,</u> <u>Kellogg G,</u> <u>Pugh BF,</u> <strong><u>Lai WKM.</u></strong></span><br />
				<span style="color:black"><b>PLoS Computational Biology.</b></span> 2022 Feb 9;18(2):e1009859. doi: 10.1371/journal.pcbi.1009859. PMID: 35139076; PMCID: PMC8863220.</p>
			</ul>
		</div>
	</div>
</div>

<footer id="footer" class="footer" role="contentinfo">
		<span>Copyright © 2022 William Lai</span>
</footer>
</div>
</div><!-- .overflow-container -->
